pillar/heroku/bootcamps.sls

```diff
@@ -134,6 +136,9 @@ heroku:
     MAX_FILE_UPLOAD_MB: 10
     NEW_RELIC_APP_NAME: Bootcamp {{ env_data.env_name }}
     NODE_MODULES_CACHE: False
+    NOVOED_API_BASE_URL: https://api.novoed.com/
+    NOVOED_API_KEY: {{ novoed.api_key }}
```

Comment:
I would change those to the following path:
vault::secret-operations/global/business_unit/novoed>data>api_key